code-style/git/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,80 @@
+### Description
+
+_[add description of work done here]_
+
+### Spec
+Designs: _[link to design if applicable]_
+
+See Story: _[/[ISSUE-NUMBER]_
+
+### Validation
+_[delete anything irrelevant to this PR]_
+
+* [ ] This PR has visual elements, so it was reviewed by a designer.
+* [ ] This PR has code changes, and our linters still pass.
+* [ ] This PR effects production code, so it was browser tested (see below).
+* [ ] This PR has new JavaScript, so new tests were added or updated, and they pass.
+* [ ] This PR has new SCSS functions, mixins, or variables, so Sass Tests were added or updated, and they pass.
+* [ ] This PR has copy changes, so copy was proofread and approved.
+* [ ] The content of this PR requires documentation, so we added a detailed description of component purpose, requirements,  quirks, and instructions for use by designers and developers. Along with accessibility information if pertinent.
+
+#### To Validate
+  1. Check that this build has not failed on Jenkins.
```

Comment:
Replace `Jenkins` with CircleCI, Codeclimate, Snyk, etc. If you wanna say something like _Make sure all PR Checks passed_ that might work too.